libs/core/src/lib/action-sheet/action-sheet-mobile/action-sheet-mobile.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject, TemplateRef } from '@angular/core';
+import { ACTION_SHEET_COMPONENT, ActionSheetInterface } from '../action-sheet.interface';
+
+@Component({
+  selector: 'fd-action-sheet-mobile',
+  templateUrl: './action-sheet-mobile.component.html',
+  styleUrls: ['./action-sheet-mobile.component.scss']
+})
+export class ActionSheetMobileComponent {
+
+    /** @hidden
+     * TODO
+     * For internal usage
+     */
+    childContent: {
+        actionSheetTemplate: TemplateRef<any>
+    } = null;
+
+    constructor(
+        @Inject(ACTION_SHEET_COMPONENT) private _actionSheetComponent: ActionSheetInterface
+    ) {}
```

Comment:
This file is not used. Mobile mode should behave same as dialog or notification, just to keep it out of DOM. IT should be appended to body, to cover whole page, when it's opened. That's you need it as dynamic component